mongodbatlas/data_source_mongodbatlas_data_lake_pipeline_test.go

```diff
@@ -36,6 +38,25 @@ func TestAccDataSourceClusterRSDataLakePipeline_basic(t *testing.T) {
 	})
 }
 
+func testAccCheckMongoDBAtlasDataLakeDestroy(s *terraform.State) error {
+	conn := testAccProviderSdkV2.Meta().(*MongoDBClient).Atlas
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "mongodbatlas_data_lake" {
+			continue
+		}
```

Comment:
Not sure if there is value in preserving this function given that it looks for the resource mongodbatlas_data_lake which is removed

Reply:
good point!

Reply:
Sorry for not stating this before, we could maintain the checkDestroy method but calling `conn.DataLakePipeline.Get(ctx, projectID, name)` to check that the data lake pipeline resource is destroyed, which is relevant for this test

Reply:
Gotcha, I was assuming it wasn't needed because I think we weren't doing it, but let me look into it

Reply:
@AgustinBettati changes done.